src/main.h

```diff
@@ -51,6 +51,37 @@ inline bool MoneyRange(int64_t nValue) { return (nValue >= 0 && nValue <= MAX_MO
 /** Threshold for nLockTime: below this value it is interpreted as block number, otherwise as UNIX timestamp. */
 static const unsigned int LOCKTIME_THRESHOLD = 500000000; // Tue Nov  5 00:53:20 1985 UTC
 
+// returns percentage reward per year
+inline int64_t GetCoinYearReward(int nDescalation) {
+    int nCurrentSupply = pindexBest->nMoneySupply;
+
+    int softForkBlockActivation = 900000; // TO BE DETERMINED
+
+    // if not yet reaching activation block
+    // or if current block is divisible by 1200
+    if (nBestHeight % 1200 || nBestHeight < 900000) {
```

Comment:
i want to check here also if 

- at supply of 10b we do divisibility by 900, which increases percentage chance of 1200% reward from 8,3% to 12.5% or so
- at 15b divisibility by 750, which is 15% or so.

same below within `GetMinStakeAge`